app/models/taxonomy_todo.rb

```diff
@@ -1,4 +1,11 @@
 class TaxonomyTodo < ApplicationRecord
   belongs_to :content_item
```

Comment:
Should a Todo behave like a ContentItem with Terms that belongs to a Project?

Maybe it should transparently delegate to all of ContentItem's property getters?